pkgs/development/libraries/webkitgtk/2.14.nix

```diff
@@ -37,24 +61,40 @@ stdenv.mkDerivation rec {
   cmakeFlags = [
   "-DPORT=GTK"
   "-DUSE_LIBHYPHEN=0"
-  "-DENABLE_GLES2=ON"
-  ];
+  ] ++ (if stdenv.isDarwin then [
```

Comment:
Use of `stdenv.lib.optionals (stdenv.isDarwin) [...]` is a much nicer syntax that avoids parenthesis nesting and plays well with additions for other platforms.